components/network/autossh/Makefile

```diff
@@ -12,25 +12,23 @@
 # Copyright 2016 Jim Klimov
 #
 
+BUILD_BITS=32
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		autossh
 COMPONENT_FMRI=		network/autossh
 COMPONENT_CLASSIFICATION=	System/Security
-COMPONENT_VERSION=	1.4e
-IPS_COMPONENT_VERSION=	1.4.5
+COMPONENT_VERSION=	1.4g
+IPS_COMPONENT_VERSION=	1.4.7
 COMPONENT_LICENSE=	BSD-like
-COMPONENT_LICENSE_FILE=	autossh.license
 COMPONENT_PROJECT_URL=	http://www.harding.motd.ca/$(COMPONENT_NAME)
 COMPONENT_SUMMARY=	autossh tunnel maintainer
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tgz
-COMPONENT_ARCHIVE_HASH=	sha256:9e8e10a59d7619176f4b986e256f776097a364d1be012781ea52e08d04679156
+COMPONENT_ARCHIVE_HASH=	sha256:5fc3cee3361ca1615af862364c480593171d0c54ec156de79fc421e31ae21277
 COMPONENT_ARCHIVE_URL=	http://www.harding.motd.ca/$(COMPONENT_NAME)/$(COMPONENT_ARCHIVE)
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+include $(WS_MAKE_RULES)/common.mk
```

Comment:
I wonder whether TEST_TARGET= $(NO_TESTS) should be used here.

Reply:
Apparently so...

Reply:
Done.